### PR TITLE
build(npm-publish): Publish npm spectral ruleset

### PR DESCRIPTION
**Pull request by leubedane**

### CODE REVIEW

**Comment by christiansiegel** [comment]
Resolves #5

File: package.json [hunk 1]
```diff
@@ -1,7 +1,8 @@
 {
   "name": "spectral-ruleset",
```
Comment (christiansiegel) [inline package.json:2]: I think we should prefix that with `@baloise`? (see also https://github.com/baloise/design-system/blob/master/packages/components/package.json#L2)
Maybe @hirsch88 can give some input here?

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -17,3 +21,8 @@ jobs:
 
       - name: Test
         run: npm run test
+
+      - name: Publish
```
Comment (christiansiegel) [inline .github/workflows/ci.yml:25]: Do we really want to publish PRs or _only_ master?

Comment (leubedane) [reply .github/workflows/ci.yml]: @christiansiegel I changed it to a new release job.

File: package.json (hunk 1 above)
Comment (leubedane) [reply package.json]: I added baloise- prefix. I can also use @baloise

Comment (leubedane) [reply package.json]: done

**Review by christiansiegel — APPROVED** [review]
lgtm

**Pull request merged**